_rules/SC2-4-2-page-has-title.md

```diff
@@ -22,19 +22,19 @@ authors:
 
 ### Applicability
 
-The rule applies to any page where the root element is an `html` element, and which is not embedded in another page.
+The rule applies to any page where the [root](https://www.w3.org/TR/dom41/#concept-tree-root) element is an `html` element, and which is not embedded in another page.
```

Comment:
The root of a document will always be the document itself so no valid HTML documents will ever match this applicability. The term we're looking for is "document element": https://www.w3.org/TR/dom/#document-element